src/templates/EditorialNewsletter/index.js

```diff
@@ -0,0 +1,251 @@
+            },
+            {
+              matchMdast: matchZone('FIGURE'),
+              component: Figure,
+              editorModule: 'figure',
+              editorOptions: {
+                afterType: 'PARAGRAPH',
+                pixelNote:
+                  'Anzeigegrössen: 1200x und 600x (proportionaler Schnitt)',
+                insertButtonText: 'Bild',
+                captionRight: true,
+                sizes: [
+                  { label: 'Gross', props: { float: undefined } },
+                  { label: 'Left', props: { float: 'left' } },
+                  { label: 'Right', props: { float: 'right' } }
+                ]
+              },
+              rules: [
+                {
+                  matchMdast: matchImageParagraph,
+                  component: Image,
+                  props: node => ({
+                    src: node.children[0].url,
+                    alt: node.children[0].alt
+                  }),
+                  editorModule: 'figureImage',
+                  isVoid: true
+                },
+                figureCaption
+              ]
+            },
+            {
+              matchMdast: matchZone('QUOTE'),
+              component: Blockquote,
+              editorModule: 'quote',
+              editorOptions: {
+                insertButtonText: 'Zitat'
+              },
+              rules: [
+                {
+                  matchMdast: (node, index, parent) =>
+                    matchParagraph(node) &&
+                    (index === 0 || !matchLast(node, index, parent)),
+                  component: BlockquoteText,
+                  editorModule: 'paragraph',
+                  editorOptions: {
+                    type: 'QUOTEP',
+                    placeholder: 'Zitat'
+                  },
+                  rules: [paragraph]
+                },
+                {
+                  matchMdast: (node, index, parent) =>
+                    matchParagraph(node) && matchLast(node, index, parent),
+                  component: BlockquoteSource,
+                  editorModule: 'paragraph',
+                  editorOptions: {
+                    type: 'QUOTECITE',
+                    placeholder: 'Quellenangabe / Autor'
+                  },
+                  rules: [paragraph]
+                }
+              ]
+            },
+            {
+              matchMdast: matchType('list'),
+              component: List,
+              props: node => ({
+                data: {
+                  ordered: node.ordered,
+                  start: node.start
+                }
+              }),
+              editorModule: 'list',
+              rules: [
+                {
+                  matchMdast: matchType('listItem'),
+                  component: ListItem,
+                  editorModule: 'listItem',
+                  rules: [paragraph]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+
+export default schema
```

Comment:
add a new line with `module.exports = schema` here to avoid the `.default` in the backend